EIPS/eip-4762.md

```diff
@@ -181,29 +168,27 @@ For `i` in `0 ... (len(code)+30)//31`.
 For a transaction, make these access events:
 
 ```
-(tx.origin, 0, VERSION_LEAF_KEY)
-(tx.origin, 0, BALANCE_LEAF_KEY)
-(tx.origin, 0, NONCE_LEAF_KEY)
-(tx.origin, 0, CODE_SIZE_LEAF_KEY)
-(tx.origin, 0, CODE_KECCAK_LEAF_KEY)
-(tx.target, 0, VERSION_LEAF_KEY)
-(tx.target, 0, BALANCE_LEAF_KEY)
-(tx.target, 0, NONCE_LEAF_KEY)
-(tx.target, 0, CODE_SIZE_LEAF_KEY)
-(tx.target, 0, CODE_KECCAK_LEAF_KEY)
+(tx.origin, 0, BASIC_DATA_LEAF_KEY)
+(tx.origin, 0, CODEHASH_LEAF_KEY)
+(tx.target, 0, BASIC_DATA_LEAF_KEY)
+```
+
+if a transaction is a successful contract creation, make this extra access event before charging the costs for the bytecode storage:
```

Comment:
could we clarify if it is before this one https://github.com/ethereum/EIPs/blob/29da7466816d023031fe0171e5adc66204fc6d87/EIPS/eip-4762.md?plain=1#L152 ?

Reply:
do we have to access it again even if we did https://github.com/ethereum/EIPs/blob/29da7466816d023031fe0171e5adc66204fc6d87/EIPS/eip-4762.md?plain=1#L72 ?